openedx/core/lib/block_structure/api.py

```diff
@@ -0,0 +1,46 @@
+"""
+Contains some functions that had been defined in lms/djangoapps/course_blocks/api.py, but are actually
+needed in both lms and cms.
+"""
+from django.core.cache import cache
+from openedx.core.lib.block_structure.manager import BlockStructureManager
+from xmodule.modulestore.django import modulestore
```

Comment:
This file needs to be in the openedx/core/djangoapps directory since it depends on django.